suite-native/atoms/src/Loader.tsx

```diff
@@ -0,0 +1,31 @@
+import { ActivityIndicator } from 'react-native';
+import React from 'react';
+
+import { useNativeStyles } from '@trezor/styles/';
+import { Color } from '@trezor/theme/';
```

Comment:
drop the `/` at the end

Reply:
fixed https://github.com/trezor/trezor-suite/pull/7563/commits/fe0aab6bcd952a8907bcf684ef9b6db00986ed78